### ISSUE TITLE
make install error

### ISSUE THREAD

**Issue by Ayanashi** [pkolano/savors#2]
makeinstall -d /usr/bin
install -d /usr/lib/savors
install -d /usr/man/man1
install -d /usr/man/man5
install -d /usr/man/man7
cd lib/cloud; make
make[1]: Entering directory '/home/savors/lib/cloud'
python setup.py build_ext -i
running build_ext
make[1]: Leaving directory '/home/savors/lib/cloud'
install -d /usr/lib/savors/cloud
install lib/cloud/query_integral_image.[sd]* /usr/lib/savors/cloud
install: cannot stat 'lib/cloud/query_integral_image.[sd]*': No such file or directory
make: *** [Makefile:33: cloud_install] Error 1

**Comment by pkolano** [comment]
Can you please show output of "ls -R" of lib/cloud directory?  A "make" in that directory is supposed to produce the file query_integral_image.so, which is what the install is looking for...

**Comment by zycecilia2006** [comment]
Makefile
> ...
>  31 cloud_install:
>  32         install -d $(PREFIX)lib/savors/cloud
>  33         install lib/cloud/query_integral_image.* $(PREFIX)lib/savors/cloud
>  34         install lib/cloud/wordcloud.py $(PREFIX)lib/savors/cloud
>...

I deleted "[sd]" and the error message disappeared.

**Comment by pkolano** [comment]
Note that removing the sd part will allow the make to proceed, but the cloud view will not work as this means the query_integral_image.so file required for it does not exist.  This is optional, however, as the rest will work without.  The "ls -R" would show if maybe the file was built as something else and/or if anything was built at all in the cloud dir.

**Comment by zycecilia2006** [comment]
Oh, in my system the full name of the `query_integral_image.so` file is `query_integral_image.cpython-38-x86_64-linux-gnu.so` . It is possibly more reasonable for me to modify the Makefile like this:
Makefile
> 33 install lib/cloud/query_integral_image.*.so $(PREFIX)lib/savors/cloud

Thanks!

**Comment by pkolano** [comment]
I figured the name might have some variation like that.  I fixed the two makefiles.  I used the slightly different query_integral_image*.[sd]* to make sure it will still cover the single dot case of a plain .so.  The d* part is for OSX .dylib.  I thought at one point it produced that on OSX although currently it looks like it is making just .so.  Anyway, I will close this one as it should be fixed.  Thanks.
